tools/c7n_azure/c7n_azure/resources/resource_map.py

```diff
@@ -80,6 +80,8 @@
     "azure.spring-service-instance": "c7n_azure.resources.spring.SpringServiceInstance",
     "azure.sql-database": "c7n_azure.resources.sqldatabase.SqlDatabase",
     "azure.sqldatabase": "c7n_azure.resources.sqldatabase.SqlDatabase",
+    "azure.sql-managed-instance": "c7n_azure.resources.sqlmanagedinstance.SqlManagedInstance",
+    "azure.sqlmanagedinstance": "c7n_azure.resources.sqlmanagedinstance.SqlManagedInstance",
```

Comment:
we generally just add aliases for compatibility, for new resources we don't generally go in with aliases.

Reply:
removed the second one